sql/core/src/test/scala/org/apache/spark/sql/execution/PlannerSuite.scala

```diff
@@ -1446,6 +1446,25 @@ class PlannerSuite extends SharedSparkSession with AdaptiveSparkPlanHelper {
       }
     }
   }
+
+  test("Limit and offset should not drop LocalLimitExec operator") {
+    val df = sql("SELECT * FROM (SELECT * FROM RANGE(100000) LIMIT 25 OFFSET 3) WHERE id > 1000")
```

Comment:
Can we minimize RANGE(100000)？ AFAIK, this operator is not optimized for skipping data yet

Reply:
Done!

Reply:
Thank you @n-young-db, can we also adjust the filter condition into a reasonable value in the WHERE clause?

Reply:
@yaooqinn oops; thank you!